landoapi/transactions.py

```diff
@@ -72,3 +72,17 @@ def get_raw_comments(transaction):
         PhabricatorClient.expect(comment, "content", "raw")
         for comment in PhabricatorClient.expect(transaction, "comments")
     ]
+
+
+def get_inline_comments(phab: PhabricatorClient, object_identifer: str) -> filter:
```

Comment:
The return type here is actually `Iterator[Transaction]`.

Reply:
I don't think `Transaction` is a type or a class here... Probably just a `dict`?

Reply:
It's [just an alias to `dict`](https://github.com/mozilla-conduit/lando-api/blob/main/landoapi/transactions.py#L10) but still technically correct. It's at least nice to know the structure of the `dict` is the one for a transaction. :)

In the future it would be nice to make the available values more defined (ie create some proper types for things like `Revision`, `Diff` etc that we pull from Phabricator).